src/Symfony/Component/Yaml/Tag/TaggedValue.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Yaml\Tag;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ * @author Guilhem N. <egetick@gmail.com>
+ */
+final class TaggedValue
+{
+    private $value;
+    private $tag;
+
+    /**
+     * @param mixed  $value
+     * @param string $tag
+     */
+    public function __construct($value, $tag)
```

Comment:
I personally see the tag as a complement of the value, not as the key; a value could be represented without a tag (a simple plain value).